JavaFiddle-Web/src/main/java/ru/javafiddle/web/models/FileJF.java

```diff
@@ -10,14 +10,14 @@
 
     private Integer fileId;
     private String  name;
-    private byte[]  data;
+    private String  data;
```

Comment:
No, we are. It's just JSON model. In order to parse it correctly and for convenience I decided to use String. In dataBase there is still byte array.